docs/architecture/specs/id_management/README.md

```diff
@@ -0,0 +1,462 @@
+		"type": "Group"
+	}]
+}
+```
+
+In the above verifiable credential fragment we are associating access
+permissions to any holder of the credentials able to authorize the users
+“0x1234” or “0x5678” or as part of the group “Group XXX” of the domain.
+
+### Json Web Tokens (JWT)
+
+JSON Web Tokens (JWT) is a compact URL-safe means of representing claims to be
+transferred between two parties. The claims in a JSON Web Tokens are encoded as
+a JavaScript Object Notation (JSON) object that is used as the payload of a JSON
+ Web Signature (JWS) structure or as the plaintext of a JSON Web Encryption
+ (JWE) structure, enabling the claims to be digitally signed or MACed and/or
+ encrypted.
+
+JSON Web Token (JWT) is an open standard [RFC 7519](https://tools.ietf.org/html/rfc7519)
+that defines a compact and self-contained way for securely transmitting
+information between parties as a JSON Object. This information can be verified
+and trusted because it is digitally signed. JSON Web Tokens can be signed using
+a secret (with HMAC algorithm) or a public/private key pair using RSA.
+
+In the architecture design we use JWT for encapsulating the messages between the
+ different parties.
+JSON Web Tokens consist of three parts separated by dots (.), which are:
+
+* Header
+* Payload
+* Signature
+
+Therefore, a JWT typically looks like the following:
+```
+Xxxxx.yyyyy.zzzzz
+```
+
+In this architecture document, all the JWT messages sent from a client to a
+server are using the HTTP Authorization header with the Bearer scheme. Example:
+
+```http
+HTTP GET /api/v1/gateway/resource
+
+Authorization: Bearer eyJhbGciOiJIUzI1NiIXVCJ9TJV...r7E20RMHrHDcEfxjoYZgeFONFh7HgQ
+```
+
+#### JWT Structure
+
+The header contains the metadata for the token and at a minimal contains the
+type of the signature and/or encryption algorithm:
+
+* "typ" - the type of the token, which is JWT
+* "alg" - the hashing algorithm such as ES256. We are using ECDSA with SHA256.
+* "cty" - Header Parameter defined by JSON Web Signature and JSON Web Encryption
+ is used by this specification to convey structural information about the JWT.
+
+In our case the header is gonna look like this:
+```json
+{
+  "typ": "JWT"
+  "alg": "ES256",
+  "cty": "arbitrary"
+}
+```
+
+#### JWT Payload
+
+The payload contains the claims. Claims are statements about an entity
+(typically, the user) and additional metadata. There are three types of claims:
+
+* Reserved claims: These are a set of predefined claims, which are not mandatory
+  but recommended, thought to provide a set of useful, interoperable claims.
+* Public claims: These can be defined at will by those using JWTs. But to avoid
+  collisions they should be defined in the IANA JSON Web Token Registry or be
+  defined as a URI that contains a collision resistant namespace.
+* Private claims: These are the custom claims created to share information
+  between parties that agree on using them.
+
+In our case the payload will looks like this:
```

Comment:
maybe its worth defining what the payload fields are the same way it was done for the header